errors.go

```diff
@@ -60,6 +60,18 @@ func EnqueueUniqueJob(ctx context.Context, queue Queue, job Job) error {
 	return errors.WithStack(err)
 }
 
+// EnqueueManyUniqueJobs is a generic wrapper for adding jobs to a queue (using the
+// PutMany() method) ignoring duplicate job errors.
```

Comment:
Can we describe more explicitly how it behaves if there's a mix of duplicate and non-duplicate jobs amongst all of the enqueued ones? And what happens if some jobs hit a duplicate job error and others hit some other kind of error?

Reply:
I wasn't sure what more to add here. The comment does say that `EnqueueManyUniqueJobs` ignores duplicate job errors which implies the rest, I think.
Though if you had something specific in mind I can definitely add it.

Reply:
Hmm, I think the question was more about how this interacts with each queue's individual implementation choice for handling a mix of duplicate and non-duplicate job errors. I would be okay with a comment as simple as something like a little warning saying since this is enqueueing multiple jobs and each queue handles mixes of errors differently, you may get different behavior depending on which queue you're using.

My main concern is that the remote queue is the special behavior in this case (it has a precedence order when it comes to deciding which error type to return) and the other local queues all behave the same way (i.e. they squash all the duplicate job errors into a generic `catcher.Resolve()` untyped error, so they'll never return a duplicate job error), and having each implementation do its own thing means just have to remember what each implementation does (alternatively, if we could make them behave the same as the remote queue does with regards to mixes of errors, then we wouldn't need all the implementation-specific comments).

Reply:
I think they should all be consistent now.
Idk what you think about where to spell out the behavior in a comment. For now it's on the WriteError's `Cause`